dish_grpc_mqtt.py

```diff
@@ -102,7 +136,9 @@ def cb_add_item(key, val, category):
             if not "dish_{0}".format(category) in data:
                 data["dish_{0}".format(category)] = {}
```

Comment:
Is it OK to leave an empty dict if the follow logic always winds up being excluded due to NaN? I guess this is dependent on whatever is consuming the the JSON?

Reply:
I don't see an issue.  For dishy, only one or two values to do with obstructions are ever returned as NaN just after boot, so there is never a case where the entire data set is NaN (eg software revision will never be NaN).

Reply:
Fair enough.